packages/babel-plugin-jest-hoist/src/index.js

```diff
@@ -10,9 +10,48 @@
 
 const JEST_GLOBAL = {name: 'jest'};
 
+const idVisitor = {
+  ReferencedIdentifier(path) {
+    this.ids.push(path);
+  },
+};
+
 const FUNCTIONS = {
   mock: {
-    checkArgs: args => args.length === 1 && args[0].isStringLiteral(),
+    checkArgs: args => {
+      if (args.length === 1) {
+        return args[0].isStringLiteral();
+      } else if (args.length === 2) {
+        const ids = [];
+        args[1].traverse(idVisitor, {ids});
+
+        for (var i = 0; i < ids.length; i++) {
+          const id = ids[i];
+          
+          let found = false;
+          let scope = id.scope;
+          
+          while (scope !== args[1].scope) {
+            if (scope.bindings[id.node.name]) {
+              found = true;
```

Comment:
this should just `return true`. That way we don't need the `found` variable at all.